src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/DefaultTranscriptsList.tsx

```diff
@@ -50,7 +50,7 @@ type ProductGeneratingContext = {
   cds: Pick<FullCDS, 'relative_start' | 'relative_end'>;
 };
 type Transcript = DefaultTranscriptListItemProps['transcript'] & {
-  so_term: FullTranscript['so_term'];
+  biotype: FullTranscript['metadata']['biotype'];
```

Comment:
Huh?

Reply:
changed to Pick2 from full transcript :-)